### PR TITLE
moved to 3.2.1/stable hadoop version

### PR DESCRIPTION
**Pull request by nevmmv**

### CODE REVIEW

**Comment by kosinsky** [comment]
Any plans to merge this PR and publish images for 3.2.x?

**Comment by gmouchakis** [comment]
Hi @kosinsky. I'll merge this and publish the images soon!

File: datanode/Dockerfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ MAINTAINER Ivan Ermilov <ivan.s.ermilov@gmail.com>
 
 HEALTHCHECK CMD curl -f http://localhost:9864/ || exit 1
 
-ENV HDFS_CONF_dfs_datanode_data_dir=file:///hadoop/dfs/data
```
Comment (gmouchakis) [inline datanode/Dockerfile]: @nevmmv why remove the file// prefix? did this cause any issues in version 3.2.1?

Comment (nevmmv) [reply datanode/Dockerfile]: @gmouchakis Yes. It did cause some error.

Comment (gmouchakis) [reply datanode/Dockerfile]: @nevmmv do you remember the error? 
I've just tried it and it works with and without `file://`. Also in hadoop documentation (https://hadoop.apache.org/docs/r3.2.1/hadoop-project-dist/hadoop-hdfs/hdfs-default.xml) the `file://` prefix is used in `dfs.namenode.name.dir` and `dfs.datanode.data.dir` so I'm thinking that maybe we should keep it this way.

Comment (nevmmv) [reply datanode/Dockerfile]: @gmouchakis This params was overrided in hadoop.env . Did you seen it?

Comment (gmouchakis) [reply datanode/Dockerfile]: @nevmmv sorry, I haven't and I cannot find where it is overrided. hadoop.env does not contain the related entries and when I check the created containers it does not seem to get overrided.

Comment (nevmmv) [reply datanode/Dockerfile]: @gmouchakis I will check it and fix.

Comment (nevmmv) [reply datanode/Dockerfile]: @gmouchakis You were right, it works with prefix `file://`. I fixed this.

**Comment by gmouchakis** [comment]
The images are ready now. Thank you @nevmmv and @kosinsky.

**Pull request merged**